molecule/api.py

```diff
@@ -9,10 +13,13 @@
 @lru_cache()
 def molecule_drivers(as_dict=False):
 
-    plugins = {
-        entry_point.name: entry_point.load()
-        for entry_point in pkg_resources.iter_entry_points('molecule_driver')
-    }
+    plugins = {}
+    for entry_point in pkg_resources.iter_entry_points('molecule_driver'):
+        try:
+            plugins[entry_point.name] = entry_point.load()
+        except Exception as e:
```

Comment:
I think this might cause some unintended consequences. I think we should limit this exception to "KeyError".

Reply:
@cloudnull In fact this piece of code is also present in a separated review and I need it for lots of exceptions, what happens is that pkg_resources can throw a very wide range of exceptions, but redoing the plugin loading will be subject of another change.

Reply:
understood. thanks for the explanation.